core/modules/system/system.module

```diff
@@ -2633,6 +2639,11 @@ function system_block_info() {
       // Names from menu_list_system_menus() need to be translated through t().
       'info' => check_plain(t($custom_title)),
       'description' => t('A list of links for the menu: %title.', array('%title' => t($custom_title))),
+      // Helper, non-standard key to distinguish the menu blocks from other
+      // types of blocks provided by the System module.
+      // @todo: should this be provided by a system_layout_context_info()
+      // implementation instead or be a 'required contexts' key?
+      'type' => 'menu',
```

Comment:
This would be an API change. Let's not do this. Instead the 'Menu:' prefix in the name.